### ISSUE TITLE
Have you solved this issue？

### ISSUE THREAD

**Issue by huxycn** [takiyu/portrait_matting#22]
When I use the command :
        $python scripts/train.py -mode  mat , 
my loss curve does not drop. Have you solved this problem?

![5](https://user-images.githubusercontent.com/42377340/50737507-2831ce80-1205-11e9-81ca-8acc33246c8b.png)

> 我已确保使用正确的值发送正确的数据。
> Alpha遮罩 - [0.255] 
> 所有重量值均为零。

When I use the command :
        $python scripts/train.py -mode  mat , 
my loss curve does not drop. Have you solved this problem?

_Originally posted by @kkkmax in https://github.com/takiyu/portrait_matting/issues/7#issuecomment-451747089_

**Comment by kkkmax** [comment]
you need a pretrain model

**Comment by Maru-chan511** [comment]
@kkkmax where is the pretrained model?

**Comment by vlordier** [comment]
@kkkmax do you mean that to converge, we need a pre-trained FCN8s like one found here https://github.com/wkentaro/fcn and then set it to `--pretrained_fcn8s` ?
Can you elaborate please ?

**Comment by vlordier** [comment]
Answer to self, for others :

This is the sort of accuracy we get using the pre-trained FCN8 from the link above.
![accuracy](https://user-images.githubusercontent.com/5443125/64419747-e3b14800-d09d-11e9-8b0d-94bd829aec7a.png)
